code/modules/organs/organ.dm

```diff
@@ -406,4 +406,4 @@ var/list/organ_cache = list()
 		return 0
 	if(robotic && robotic < ORGAN_LIFELIKE)	//Super fancy humanlike robotics probably have sensors, or something?
 		return 0
-	return 1
\ No newline at end of file
+	return 1
```

Comment:
Don't randomly add this in Polaris files. It will conflict whenever they add anything to the file because it's now 'longer' than this change.